src/models/userModel.ts

```diff
@@ -0,0 +1,4 @@
+import { type z } from 'zod';
+import { type schemaUser } from './userSchema';
+
+export type UserBodyType = z.infer<typeof schemaUser>;
```

Comment:
You can import the user type from prisma

Reply:
Type from prisma contains a mandatory ID, which I don't need in all methods.

Reply:
Have you heard about a generic type omit?
https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys

Reply:
Yeah, heard of it, but didn't think of it). Thanks!